.github/workflows/ci-integration-test-artifacts-multiple-folders.yml

```diff
@@ -0,0 +1,42 @@
+name: CI - integration test artifacts multiple folders
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  main:
+    name: Nx Cloud - Main Job
+    uses: ./.github/workflows/nx-cloud-main.yml
+    with:
+      working-directory: ./integration-tests/npm
+      parallel-commands-on-agents: |
+        npx nx run react-app:test --ci --coverage
+        npx nx run react-app:build
+      artifacts: |
+        ./integration-tests/npm/dist/
+        ./integration-tests/npm/coverage/
+      artifactsName: custom-artifacts-name
```

Comment:
We use `dash-case` for all other inputs, please align i.e. `artifacts-name`.

Additionally, if there is agreement that `artifacts-name` is going to be supported, then I wold vote to change `artifacts` to `artifacts-path` to make it clearer what it corresponds to